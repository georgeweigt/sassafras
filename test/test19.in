data
input y
datalines
1
2
3
4
5
6
;
proc means clm
