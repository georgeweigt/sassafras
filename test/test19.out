                    Variable     95% CLM MIN     95% CLM MAX
                    Y                  1.537           5.463

